src/ctia/features_service.clj

```diff
@@ -0,0 +1,32 @@
+(ns ctia.features-service
+  (:require [puppetlabs.trapperkeeper.core :as tk]
+            [clojure.string :as string]))
+
+(defprotocol FeaturesService
+  "Service to read configuration properties under [:ctia :features] key in config.
+
+  If an option like this: `ctia.features.disable=asset,actor,sighting` is added
+  to the properties file (ctia-default.properties) - http routes for Asset,
+  Actor, and Sighting entities would be disabled and their respective contexts
+  would not be visible in Swagger console."
+  (feature-flags [this]
+    "Returns all feature flags defined in the config")
+  (disabled? [this key]
+    "Returns true if the given entity key is marked as Disabled in properties config")
+  (enabled? [this key]
```

Comment:
Maybe I'd offer only `enabled?` as an API since consumers can always do the `not` themselves :)

(and I'd pick enabled? over disabled? for avoiding double negatives)